### DISCUSSION TITLE
MAX30100 in Tasmota.

### DISCUSSION THREAD

**Discussion by TomaszSz7** [arendst/Tasmota#18289] [Ideas]
Hello 

Anyone can help with the implementation of MAX30100 sensor ?
The driver for this device should be loaded. Unfortunately I do not have enough knowledge to do this.

Communication is via I2C

Drivers -> https://github.com/xcoder123/MAX30100

thanks in advance for your support.

BR
Tom

**Comment by barbudor** [comment]
> The driver for this device should be loaded

What does that mean considering that there is currently no driver for that chip in Tasmota ?

What you are pointing is an Arduino library that needs actually to be wrapped up into a Tasmota driver.
Some coding required
Must read : https://tasmota.github.io/docs/For-Developers/
As well as other existing drivers for I2C sensors
